.gitlab-ci.yml

```diff
@@ -46,13 +46,25 @@ test_big_submission:
   tags:
     - ingest-integration-tests
 
+test_bulk_update:
+  stage: test
+  script:
+     - python -m unittest tests.test_ingest.TestRun.test_bulk_update
+  only:
+    - dev
+    - staging
+    - add_test_for_bulk_update
```

Comment:
please don't forget to change later! i also think it's okay to include this test in production, it's not exporting anything so it should be fine

Reply:
ah, yep, tx. I remembered it last night, but it was to late and wanted to do today, then I forgot. Many thanks. I am going to changed it back now.

Reply:
Done ✅